android/Cargo.toml

```diff
@@ -13,10 +13,10 @@ authors.workspace = true
 [dependencies]
 maplibre = { path = "../maplibre", features = ["thread-safe-futures"] }
 maplibre-winit = { path = "../maplibre-winit", version = "0.1.0"  }
-env_logger = "0.9.0"
+env_logger = "0.10.0"
 log = "0.4.17"
 ndk-glue = "0.7.0"  # version is required by winit
-jni = "0.19.0"
+jni = "0.20.0"
```

Comment:
Did you test the android example? If not we might just keep the old version.

Reply:
sorry indeed, I went a bit too far.
I haven’t yet setup the android toolchain. I switched it bach to 0.19